modules/openxr/extensions/openxr_ml2_controller_extension.cpp

```diff
@@ -0,0 +1,78 @@
+/**************************************************************************/
+/*  openxr_ml2_controller_extension.cpp                                   */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#include "openxr_ml2_controller_extension.h"
+#include "../action_map/openxr_interaction_profile_meta_data.h"
+
+// ML2 controllers are not part of the included OpenXR spec at the time of
```

Comment:
Seeing we simply need to update the thirdparty inclusion of OpenXR headers to fix this, I suggest we do that. @akien-mga where do we stand on updating thirdparty libraries now that we're so close to RC1? If thats no problem I'll prepare a PR for moving us to OpenXR 1.0.26

Reply:
Yeah you can update the OpenXR lib, I don't expect any problem from it.

Reply:
Ok I'm not sure where the confusion has come from, but as far as I can see we are already including OpenXR 1.0.26 and our ML2 extension is in thirdparty:

https://github.com/godotengine/godot/blob/master/thirdparty/openxr/include/openxr/openxr.h#L4043

Reply:
So @rbessems I think you can just removed the `#ifndef XR_ML_ML2_CONTROLLER_INTERACTION_EXTENSION_NAME` block?

Reply:
@BastiaanOlij I've removed the ifdef's.